### PR TITLE
upgraded to Ignite 0.3.0 and fixed upgrade compatibility

### PR DESCRIPTION
**Pull request by maxkazmsft**
closes https://github.com/microsoft/seismic-deeplearning/issues/214

### CODE REVIEW

File: examples/interpretation/notebooks/Dutch_F3_patch_model_training_and_evaluation.ipynb [hunk 1]
```diff
@@ -117,17 +117,7 @@
    "outputs": [],
    "source": [
     "import numpy as np\n",
```
Comment (sharatsc) [inline examples/interpretation/notebooks/Dutch_F3_patch_model_training_and_evaluation.ipynb:119]: Looks like you need to merge earlier commits

Comment (maxkazmsft) [reply examples/interpretation/notebooks/Dutch_F3_patch_model_training_and_evaluation.ipynb:119]: looks good to me... what's the problem?

Comment (yalaudah) [reply examples/interpretation/notebooks/Dutch_F3_patch_model_training_and_evaluation.ipynb:119]: I think Sharat meant that these changes were already made in PR #247.

**Comment by maxkazmsft** [comment]
closing due to bug in distributed training - will use dev version for now

**Pull request closed without merging**